kubernetes/worker.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: ray-worker
```

Comment:
One question here. If we use deployment here, new pods will be recreated automatically when we delete the old ones. Should we use pod directly or should we co-design the scaling-down support with ray-operator.

Reply:
I think using pods directly for now is a good choice because it's most similar to how the autoscaler works on other platforms (e.g., managing EC2 instances directly). In the future we should think more carefully about the right abstraction and properties when nodes go down.

Reply:
Yeah, I think so. In this approach, maybe we do not need an operator to manage the cluster in K8s. We can use the autoscaler as the operator.

PS, what will happen if there is a node down? Will the autoscaler create a new one?